docs(ct): add brief information about CSS modules into FAQ (#34677)
microsoft/playwright: docs/src/test-components-js.md

});
```

> ### I have a component that imports CSS. How can I configure Vite to handle CSS files?

Reviewer: ```suggestion
### How do I use CSS imports?
```

> If you have a component that imports a CSS file, Vite will handle it automatically. You can also use CSS Pre-processors such as Sass, Less, or Stylus and Vite will handle them as well without any additional configuration. However, corresponding CSS Pre-processor needs to be installed.

Reviewer: ```suggestion
If you have a component that imports CSS, Vite will handle it automatically. You can also use CSS pre-processors such as Sass, Less, or Stylus, and Vite will handle them as well without any additional configuration. However, corresponding CSS pre-processor needs to be installed.
```

Author: I like to have it lowercase as you suggest it, but Vite have in uppercase in the docs. But changed as you wish.

> To use CSS modules, you must rename your `.css` files to `.module.css` and rename your imports to include the `.module` part, otherwise Vite will throw an error `RollupError: "default" is not exported by "<file>.css"`.  This requirement applies to files using CSS Pre-processors as well.

Reviewer: ```suggestion
Vite has a hard requirement that all CSS Modules are named `*.module.[css extension]`. If you have a custom build config for your project normally and have imports of the form `import styles from 'styles.css'` you must rename your files to properly indicate they are to be treated as modules. You could also write a Vite plugin to handle this for you.
```
